.github/workflows/pr-build.yaml

```diff
@@ -0,0 +1,63 @@
+name: PR Build
+
+on:
+    pull_request:
+        branches: [main]
+
+jobs:
+    build-integrationtests:
+        name: Building the Integration Tests Repository
+        runs-on: ubuntu-latest
+        
+        steps:
+            - name: Checkout Code
+              uses: actions/checkout@v4
+            
+            - name: Setup JDK 
+              uses: actions/setup-java@v4
+              with:
+                java-version: '11'
+                distribution: 'semeru'
+                cache: maven
+          
+            - name: Print githash
+              run: |
+                  echo $GITHUB_SHA > ./inttests.githash
+
+            - name: Setup Gradle
+              uses: gradle/actions/setup-gradle@v3
+              with:
+                gradle-version: 6.9.2
+            
+            - name: Build Integration Tests Gradle source code
+              run: |
+                gradle -b galasa-inttests-parent/build.gradle publish publishToMavenLocal \
+                -PsourceMaven=https://development.galasa.dev/gh/maven-repo/obr \
+                -PcentralMaven=https://repo.maven.apache.org/maven2/ \
+                -PtargetMaven=${{github.workspace}}/repo
+            
+            - name: Building Integration Tests Maven source code
+              run: |
+                  mvn -f galasa-inttests-parent/dev.galasa.inttests.obr/pom.xml deploy -Dgpg.skip=true \
+                  -Dgalasa.source.repo=https://development.galasa.dev/gh/maven-repo/obr \
+                  -Dgalasa.central.repo=https://repo.maven.apache.org/maven2/ \
+                  -Dgalasa.release.repo=file:${{ github.workspace }}/repo \
+                  -Dgalasa.skip.deploytestcatalog=true \
+                  -Dgalasa.skip.bundletestcatalog=false \
+                  --batch-mode --errors --fail-at-end \
+                  --settings ${{github.workspace}}/settings.xml
+
+            
+            - name: Build Integration Tests image for testing
+              uses: docker/build-push-action@v5
+              with:
+               context: .
+               file: dockerfiles/dockerfile.inttests
+               load: true
+               tags: intests:test
+               build-args: |
+                baseVersion=latest
+                dockerRepository=harbor.galasa.dev

```

Comment:
Can you please change this to `ghcr.io`?

Reply:
done